tests/cli/test_mixed_configurations.py

```diff
@@ -35,40 +47,43 @@ def test_coexisting_configurations(click_runner,
     # Setup
     #
 
+    if custom_filepath.exists():
+        shutil.rmtree(custom_filepath, ignore_errors=True)
+    assert not custom_filepath.exists()
+
     # Parse node addresses
     alice, ursula, another_ursula, felix, staker, *all_yall = testerchain.unassigned_accounts
 
     envvars = {NUCYPHER_ENVVAR_KEYRING_PASSWORD: INSECURE_DEVELOPMENT_PASSWORD,
                'NUCYPHER_FELIX_DB_SECRET': INSECURE_DEVELOPMENT_PASSWORD}
 
     # Future configuration filepaths for assertions...
-    public_keys_dir = os.path.join(custom_filepath, 'keyring', 'public')
-    known_nodes_dir = os.path.join(custom_filepath, 'known_nodes')
+    public_keys_dir = custom_filepath / 'keyring' / 'public'
+    known_nodes_dir = custom_filepath / 'known_nodes'
 
     # ... Ensure they do not exist to begin with.
-    assert not os.path.isdir(public_keys_dir)
-    assert not os.path.isfile(known_nodes_dir)
 
     # No keys have been generated...
-    with pytest.raises(FileNotFoundError):
-        assert len(os.listdir(public_keys_dir)) == 0
+    assert not public_keys_dir.exists()
 
     # No known nodes exist...
-    with pytest.raises(FileNotFoundError):
-        assert len(os.listdir(known_nodes_dir)) == 0
+    assert not known_nodes_dir.exists()
 
     # Not the configuration root...
     assert not os.path.isdir(custom_filepath)
 
+    # ... nothing
```

Comment:
I'm not sure which is better commentary here, the comment or the `None`.

Reply:
😂